README.md

```diff
@@ -1,2 +1,51 @@
 # opengemini-client-java
-openGemini java client(sdk)
+
+opengemini-client-java is a Java client for OpenGemini. 
+
+English | [简体中文](README_CN.md)
+
+OpenGemini is an open-source time series database, find more about OpenGemini at https://github.com/openGemini/openGemini
+## Characteristic
+
+- This project is implemented independently by multiple components, which can be referenced separately according to different requirements.
+- This project is implemented in line with other SDKs such as opengemini-client-go and opengemini-client-python, adhering to the same API layer philosophy, resulting in a very similar user experience for different users.
+
+| Sdk Components | Runtime Minimum JDK Version| Recommended Scenario | Notice |
+| --------------------------------- | ------------------- | ------------------------------------ | ------------------------------------------------------------ |
```

Comment:
jdk9 is not a LTS version, only jdk8、jdk11、jdk17、jdk21 and incoming jdk25 is LTS